### PR TITLE
Updated SDK to Network-2018-04-01 branch

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
## Description
Updated Network SDK to Network-2018-04-01 swagger branch.
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by MikhailTryakhov** [comment]
+@dsgouda to help with review of April branch

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>18.0.1-preview</Version>
+		<Version>19.0.1-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: 1.  Versioning should start from  19.0.0-preview
2. `PackageReleaseNotes` must be updated here

@shahabhijeet Doesn't look like this PR has any breaking changes but the spec from which this code was generated was from a new version, should we still mark this as a major package revision?

**Review by dsgouda — CHANGES_REQUESTED** [review]
1. Code must be generated either using the `generate.ps1` or the `generate.cmd` files which should make changes to a log txt file, please commit these changes
2. Please run `msbuild build.proj /t:build /p:Scope=SDKS\Network` and commit changes made to the .props file

**Comment by MikhailTryakhov** [comment]
@dsgouda generated as you told - new commit arrived.
build command hasn't generated any changes...

**Comment by MikhailTryakhov** [comment]
+@shahabhijeet to review. Please give a try to review it today

**Review by dsgouda — APPROVED** [review]
LGTM subject to CI builds. Still not sure if this should be a major package version change, waiting on @shahabhijeet

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj [hunk 2]
```diff
@@ -7,15 +7,11 @@
...
+		<Version>19.0.0-preview</Version>
 		<PackageTags>Microsoft Azure Network management;Network;Network management;</PackageTags>    
 		<PackageReleaseNotes>
 		<![CDATA[
-			- DDoS Protection Plans
-			- Express Route Circuit Connection
-			- Express Route Provider APIs
-			- GetTopologyOperation supports query parameter
-			- Custom IPsec/IKE policy for Virtual Network Gateway point-to-site clients
+			- Made id read-only by bringing properties from shared resource
```
Comment (dsgouda) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: Which resource? please be explicit

**Comment by dsgouda** [comment]
Ready to merge when CI builds pass

**Comment by MikhailTryakhov** [comment]
@dsgouda @shahabhijeet fixed, ready for merge

**Pull request merged**